charts/victoria-metrics-cluster/values.yaml

```diff
@@ -49,6 +49,19 @@ vmselect:
   extraLabels: {}
   env: []
 
+  # Readiness & Liveness probes
+  probe:
+    readiness:
+      initialDelaySeconds: 5
+      periodSeconds: 15
+      timeoutSeconds: 5
```

Comment:
this param can confuse as it is not been used in template but defined

Reply:
understood, already fixed it)